gcp/appengine/frontend3/src/templates/vulnerability.html

```diff
@@ -3,169 +3,159 @@
         {# TODO(andrewpollock): it's permissible to have more than one repo, so handle this -#}
-        <span class="vuln-name spicy-sections-workaround">{{ affected['package']['name'] or vulnerability.repo | strip_scheme }}</span>
+        <span class="vuln-name spicy-sections-workaround">{{ affected['package']['name'] or vulnerability.repo |
+          strip_scheme }}</span>
       </h2>
       <div class="mdc-layout-grid">
-        <div class="vulnerability-package-subsection mdc-layout-grid__inner">
-          <h3 class="mdc-layout-grid__cell--span-3">
-            Package Info
-          </h3>
-          <div class="mdc-layout-grid__cell--span-9">
-            <dl>
-              <dt>Package Name</dt>
-              <dd>{{ affected['package']['name'] or 'Unknown' }}</dd>
-              <dt>Repo URL</dt>
-              {# TODO(andrewpollock): it's permissible to have more than one repo, so handle this -#}
-              <dd><a href="{{ vulnerability.repo }}">{{ vulnerability.repo }}</a></dd>
-            </dl>
-          </div>
-        </div>
         <div class="vulnerability-package-subsection mdc-layout-grid__inner">
           <h3 class="mdc-layout-grid__cell--span-3">
             Affected ranges
-            <a href="https://ossf.github.io/osv-schema/#examples" target="_blank"></a>
+            <a href="https://ossf.github.io/osv-schema/#examples" target="_blank" rel="noopener noreferrer"></a>
           </h3>
           <div class="mdc-layout-grid__cell--span-9">
             {% for range in affected['ranges'] -%}
-              <dl>
-                <dt>Type</dt>
-                <dd>{{ range['type'] -}}</dd>
-                <dt>Events</dt>
-                <dd>
-                  <div class="mdc-layout-grid__inner events">
-                    {% for event in range['events'] -%}
-                      <div class="mdc-layout-grid__cell--span-3">
-                        {{ event | event_type -}}
-                      </div>
-                      <div class="mdc-layout-grid__cell--span-9 version-value">
-                        {% set link = event | event_link -%}
-                        {% if link -%}
-                          <a href="{{ link }}"}>
-                        {% elif event | event_type == 'Introduced' and event | event_value == '0' -%}
-                          <div class="tooltip">
+            <dl>
+              <dt>Type</dt>
+              <dd>{{ range['type'] -}}</dd>
+              <dt>Events</dt>
+              <dd>
+                <div class="mdc-layout-grid__inner events">
+                  {% for event in range['events'] -%}
+                  <div class="mdc-layout-grid__cell--span-3">
+                    {{ event | event_type -}}
+                  </div>
+                  <div class="mdc-layout-grid__cell--span-9 version-value">
+                    {% set link = event | event_link -%}
+                    {% if link -%}
+                    <a href="{{ link }}" }>
```

Comment:
This `}` seems out of place...

Reply:
Well spotted, thanks! Fixed in https://github.com/google/osv.dev/pull/1483/commits/65884fcd8820292d955a1fb10880353808bc9195